.circleci/config.yml

```diff
@@ -62,7 +62,11 @@ jobs:
           command: ./test/tools/integration/provision_master.sh
       - run:
           name:  Run E2E test
-          command: go test -tags=e2e -parallel 24 -v -timeout 20m  ./test/e2e/... -identifier=$CIRCLE_BUILD_NUM
+          command: >-
```

Comment:
do you think we could export the content of this command to a file? So that I can run the test from my local machine the same way we they are run in `CI` ? 

I'm not saying we have to do this in this pull, just an idea...

Reply:
Well, the provisioning script copy the kubeconfig over to `$GOPATH/src/github.com/kubermatic/machine-controller/.kubeconfig` so  you can run the `go test -tags=e2e -parallel 24 -v -timeout 20m  ./test/e2e/...` fine locally.

The `export KUBECONFIG=...` is just to make configure `kubectl` to use the correct kubeconfig for logs in case of failure. I am a bit hesitant to write a script that changes the users environment because you are suddenly talking to a diffrent k8s cluster

Reply:
I think that you cannot change parent's shell variables from a child process. So `KUBECONFIG` will not be overwritten.